helpers/mail/mail_v3_test.go

```diff
@@ -814,3 +814,20 @@ func TestParseInvalidEmail(t *testing.T) {
 		t.Error("Expected an error to be thrown from ParseEmail")
 	}
 }
+
+func TestParseInvalidEmailLength(t *testing.T) {
```

Comment:
Please provide the happy path test, thanks!

Reply:
sure.

Reply:
@thinkingserious I have added new tests for the happy path :)

Reply:
Thank you!